src/renderer/corporate/common/common-helpers.jsx

```diff
@@ -34,9 +34,9 @@ const getEntityJurisdiction = entry => {
 		entry.identity.type === 'individual'
 			? 'http://platform.selfkey.org/schema/attribute/nationality.json'
```

Comment:
should be constants

Reply:
Yeah, this is not good. 
I had already started a refactoring on the edit member pull request that's still wip.

Reply:
Access to attribute values is refactored here: 135af86